control/tests/rlocus_test.py

```diff
@@ -48,6 +50,7 @@ def test_without_gains(self, sys):
         roots, kvect = root_locus(sys, plot=False)
         self.check_cl_poles(sys, roots, kvect)
 
+    @X11only
```

Comment:
The failures are not due to X11 but because the figure manager has no toolbar. I guess the matplorlib version/setup is different than on the Travis runners.

Same problem has always been present for the openSUSE rpm package, which I maintain. Forcing the PyQt5 backend instead of Agg helped there, but my tries (short) for Slycot on GitHub worklfows did not succeed. I just skip the two tests in python-control/Slycot#140

Reply:
Thanks for the extra info. I'd prefer not to have to factor these out.  Let me see if I can use pytest-xvfb or something similar to get it to work.

Reply:
I ended up leaving in these marks since they are useful if you are testing without X11, but also got pytest-xvfb working, so these tests aren't skipped in the GitHub Actions CI testing.